<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1431</width>
      <height>801</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab1">
        <attribute name="title">
         <string>Load Config</string>
        </attribute>
        <widget class="QToolBar" name="toolBar1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>71</width>
           <height>30</height>
          </rect>
         </property>
         <addaction name="actionLoadCfg"/>
        </widget>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>551</width>
           <height>491</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label2">
            <property name="text">
             <string>field2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit1"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit2"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit3"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label3">
            <property name="text">
             <string>field3</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QGraphicsView" name="graphicsView"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label1">
            <property name="text">
             <string>field1</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label4">
            <property name="text">
             <string>image1</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label5">
            <property name="text">
             <string>image2</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QGraphicsView" name="graphicsView_2"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Show Image</string>
        </attribute>
        <widget class="QGraphicsView" name="ImageGraphic">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>411</width>
           <height>301</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="showImageButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>92</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>show</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionLoadCfg">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Settings.ico</normaloff>:/icons/Settings.ico</iconset>
   </property>
   <property name="text">
    <string>LoadCfg</string>
   </property>
   <property name="toolTip">
    <string>Load config from file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
